checks/pinned_dependencies_test.go

```diff
@@ -515,9 +515,9 @@ func TestGitHubWorflowRunDownload(t *testing.T) {
 			filename: "testdata/github-workflow-wget-across-steps",
 			expected: scut.TestReturn{
 				Errors:        nil,
-				Score:         checker.MinResultScore,
-				NumberOfWarn:  2,
-				NumberOfInfo:  0,
+				Score:         checker.MaxResultScore,
+				NumberOfWarn:  0,
+				NumberOfInfo:  1,
```

Comment:
This is because we are no longer parsing `ksh`: https://github.com/ossf/scorecard/blob/8960533b7b38916ad1c0a235643e9468ffbdd152/checks/testdata/github-workflow-wget-across-steps#L43

Reply:
instead of ignoring the file, change the shell name in the workflow. We want to make sure this test passes with a supported shell.